lib/routes/sign.js

```diff
@@ -2,6 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var crypto = require('crypto')
+
+var TIMED_KEY = crypto.randomBytes(8)
+setInterval(function () {
+  TIMED_KEY = crypto.randomBytes(8)
+}, 1000 * 60 * 60 * 24);
```

Comment:
You'll get a different key per webhead; might need to store it in shared memcache or use something derived from a timestamp if you want to do this.  It may not be worth the effort.